angular/projects/admin-nrpti/src/app/mines/mines-records-list/mines-records-list.component.ts

```diff
@@ -413,6 +466,37 @@ export class MinesRecordsListComponent implements OnInit, OnDestroy {
     );
   }
 
+  /**
+   * Cancel adding records to collection.
+   *
+   * @memberof MinesRecordsListComponent
+   */
+  cancelAddEditCollectionRecords() {
+    const shouldCancel = confirm(
```

Comment:
Clicking cancel always routes you to the `add\edit collection` screen, rather than just clearing the "selected" collection and checked records. If you came in from the Mine record and just wanted to create a net-new collection, this would be unexpected!

Reply:
This should be fixed.  If you cancel, it should send you back to the add/edit collection page, with whatever values you started with originally.  Before, I was still sending along the latest records that you added, which isn't right.